packages/insomnia-app/app/ui/components/unit-test-selection.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import React from 'react';
+
+type Props = {
+  onChange: Function,
+  unitTest: Object,
+  selectableRequests: Object,
```

Comment:
It would be great if we could type these more explicitly.

For example, just from looking at the usage in the component
```suggestion
  onChange: () => void,
  selectedRequestId?: string,
  selectableRequests: Array<{name: string, request: { _id: string }}>,
```